plugin/src/main/cpp/classes/openxr_fb_spatial_entity.cpp

```diff
@@ -353,18 +353,20 @@ Node3D *OpenXRFbSpatialEntity::create_collision_shape() const {
 
 		return collision_shape;
 	} else if (is_component_enabled(COMPONENT_TYPE_BOUNDED_2D)) {
+		const float COLLISION_SHAPE_2D_THICKNESS = 0.1;
```

Comment:
Would be good to take the precaution and have this be a project settings.

Reply:
I've added a project setting for this, which defaults to `0.1`